### ISSUE TITLE
::first-letter in print style can cause incorrect rendering in Chrome

### ISSUE THREAD

**Issue by patrickhlauke** [h5bp/html5-boilerplate#1961]
**I'm submitting a ...**

- [x] bug report
- [ ] feature request
- [ ] other (Please do not submit support requests here (below))

Chrome currently has a bug that results in `::first-letter` being vertically misaligned under certain (seemingly unrelated) conditions. See this fairly long-standing Bootstrap bug https://github.com/twbs/bootstrap/issues/21771

There is an issue in Chrome's bug tracker - however, for some reason, the issue was recently locked/restricted (possibly because it has security implications). The bug is/was https://bugs.chromium.org/p/chromium/issues/detail?id=739800, filed a few days ago (I've asked contacts at Google if it would be possible to un-restrict the bug again).

For Bootstrap, I'm likely going to simply remove the `::first-letter` and `::first-line` selectors since BS doesn't mess with any of those (see https://github.com/twbs/bootstrap/pull/23044)

**Comment by coliff** [comment]
Interesting. I think it'd make sense to remove `::first-letter` and `::first-line` selectors from the print styles until the Chrome issue is resolved then as it does look pretty messed up. I'll open a PR. Thanks for heads-up!

**Comment by roblarsen** [comment]
Thanks @patrickhlauke and @coliff :1st_place_medal:

**Comment by roblarsen** [comment]
closed via #1962

**Comment by slackday** [comment]
Nice that this has been removed. I've been trying to fix it in projects using this build of Bootstrap. For anyone else using alpha/beta builds of Bootstrap and needs to fix this issue you can add 

```
@media print {
  *::first-letter {
    position: relative !important; 
  }
}
```

to your project stylesheet.
